CMakeLists.txt

```diff
@@ -1112,3 +1112,14 @@ FILE(STRINGS "include/openssl/opensslv.h"
 if(NOT ${BASE_VERSION_NUMBER} MATCHES ${OPENSSLV_VERSION_NUMBER})
   message( FATAL_ERROR "OPENSSL_VERSION_NUMBER in base.h and opensslv.h should match.")
 endif()
+
+# TODO: Find a way to make this automatically align with OPENSSL_VERSION_NUMBER in base.h.
+set(VERSION 1.1.1)
+
+file(GLOB OPENSSL_PKGCONFIGS "pkgconfig/*.pc.in")
+foreach(in_file ${OPENSSL_PKGCONFIGS})
+  file(RELATIVE_PATH in_file ${PROJECT_SOURCE_DIR} ${in_file})
+  string(REPLACE ".in" "" pc_file ${in_file})
+  configure_file(${in_file} ${CMAKE_CURRENT_BINARY_DIR}/${pc_file} @ONLY)
+  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${pc_file} DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
```

Comment:
for developers, will these packageconfig package names (openssl, libssl, libcrypto) conflict with locally-built OpenSSL (real OpenSSL) installations managed by packageconfig? if so, is there some place we can document the expectation of these conflicts?

Reply:
They're intentionally the same so projects consuming OpenSSL in this form can detect AWS-LC with minimal changes to their build script. We can use `PKG_CONFIG_PATH` to pass the right pkg-config artifacts to the build. I can add that somewhere around here.